src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPrices.tsx

```diff
@@ -0,0 +1,331 @@
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import FormSpacer from "@saleor/components/FormSpacer";
+import Grid from "@saleor/components/Grid";
+import Hr from "@saleor/components/Hr";
+import SingleSelectField from "@saleor/components/SingleSelectField";
+import { ProductDetails_product_productType_variantAttributes } from "@saleor/products/types/ProductDetails";
+import CardTitle from "@saleor/components/CardTitle";
+import CardSpacer from "@saleor/components/CardSpacer";
+import { ProductVariantCreateFormData } from "./form";
+
+const useStyles = makeStyles(
+  theme => ({
+    hr: {
+      marginBottom: theme.spacing(),
+      marginTop: theme.spacing(0.5)
+    },
+    hrAttribute: {
+      marginTop: theme.spacing(2)
+    },
+    label: {
+      alignSelf: "center"
+    },
+    shortInput: {
+      width: "33%"
+    }
+  }),
+  { name: "ProductVariantCreatorPrices" }
+);
+
+export type PriceOrStock = "price" | "stock";
+export interface ProductVariantCreatorPricesProps {
+  attributes: ProductDetails_product_productType_variantAttributes[];
+  currencySymbol: string;
+  data: ProductVariantCreateFormData;
+  onApplyPriceOrStockChange: (applyToAll: boolean, type: PriceOrStock) => void;
+  onApplyToAllChange: (value: string, type: PriceOrStock) => void;
+  onAttributeSelect: (id: string, type: PriceOrStock) => void;
+  onAttributeValueChange: (
+    id: string,
+    value: string,
+    type: PriceOrStock
+  ) => void;
+}
+
+const ProductVariantCreatorPrices: React.FC<ProductVariantCreatorPricesProps> = props => {
+  const {
+    attributes,
+    currencySymbol,
+    data,
+    onApplyPriceOrStockChange,
+    onApplyToAllChange,
+    onAttributeSelect,
+    onAttributeValueChange
+  } = props;
+  const classes = useStyles(props);
+  const intl = useIntl();
+
+  const attributeChoices = attributes.map(attribute => ({
+    label: attribute.name,
+    value: attribute.id
+  }));
+  const priceAttributeValues = data.price.all
+    ? null
+    : data.price.attribute
+    ? attributes
+        .find(attribute => attribute.id === data.price.attribute)
```

Comment:
Data structure is bit funky, could we move this filtering to the utils? `getAttributeValues(attribute)` maybe.